integration-test/src/test/java/org/uniprot/store/indexer/search/uniprot/CCAPSearchIT.java

```diff
@@ -133,67 +133,6 @@ void testAP() {
         assertThat(retrievedAccessions, hasItems(P48347, Q6GZN7, Q6V4H0));
     }
 
-    @Test
-    void testAPEv() {
```

Comment:
do we have test with cc_ap only?

Reply:
The test above testAP() cover this use case with cc_ap only